src/commands/lambda/instrument.ts

```diff
@@ -53,6 +58,18 @@ export class InstrumentCommand extends Command {
       return 1
     }
 
+    if (this.sourceCodeIntegration) {
+      if (!process.env.DATADOG_API_KEY && !process.env.DATADOG_KMS_API_KEY) {
```

Comment:
I think we are adding support for a secret ARN as well -- I would coordinate with @duncanista on this

Reply:
Sounds good -- maybe we should pull this out into a helper function then so we don't need to re-implement everywhere.

Reply:
Per real-life convo, never mind (this is a bit of a different use case)

Reply:
Thinking more about this, we should check to make sure that DATADOG_KMS_API_KEY is supported here as well (it might not be)

Reply:
In fact, I'm almost sure it's not

Reply:
After a discussion in person, it probably doesn't make sense to keep DATADOG_KMS_API_KEY here. Thanks for clarifying!